ModelFiles/yml/yeastGEM.yml

```diff
@@ -93747,7 +93818,187 @@
       - metanetx.reaction: MNXR122225
       - sbo: SBO:0000176
     - confidence_score: 0
+  - !!omap
+    - id: r_4587
+    - name: Ca(2+) transport 
+    - metabolites: !!omap
+      - s_0794: 1
+      - s_0796: -1
+      - s_3880: 1
+      - s_4199: -1
+    - lower_bound: -1000
+    - upper_bound: 1000
+    - gene_reaction_rule: YOL122C
+    - annotation: !!omap
+      - metanetx.reaction: MNXR96437
+      - sbo: SBO:0000655
+    - confidence_score: 1
+  - !!omap
+    - id: r_4588
+    - name: Cl(-) transport
+    - metabolites: !!omap
+      - s_3778: 1
+      - s_4200: -1
+    - lower_bound: -1000
+    - upper_bound: 1000
+    - gene_reaction_rule: YNL275W
+    - annotation: !!omap
+      - metanetx.reaction: MNXR96797
+      - sbo: SBO:0000655
+    - confidence_score: 1
+  - !!omap
+    - id: r_4589
+    - name: Cu2(+) transport
+    - metabolites: !!omap
+      - s_0794: 1
+      - s_0796: -1
+      - s_4019: 1
+      - s_4201: -1
+    - lower_bound: -1000
+    - upper_bound: 1000
+    - gene_reaction_rule: YOL122C
+    - annotation: !!omap
+      - metanetx.reaction: MNXR126350
+      - sbo: SBO:0000655
+    - confidence_score: 1
+  - !!omap
+    - id: r_4590
+    - name: Mn(2+) transport
+    - metabolites: !!omap
+      - s_0794: 1
+      - s_0796: -1
+      - s_3801: 1
+      - s_4202: -1
+    - lower_bound: -1000
+    - upper_bound: 1000
+    - gene_reaction_rule: YGR191W
+    - annotation: !!omap
+      - metanetx.reaction: MNXR101669
+      - sbo: SBO:0000655
+    - confidence_score: 1
+  - !!omap
+    - id: r_4591
+    - name: Zn(2+) transport
+    - metabolites: !!omap
+      - s_0794: 1
+      - s_0796: -1
+      - s_3822: 1
+      - s_4203: -1
+    - lower_bound: -1000
+    - upper_bound: 1000
+    - gene_reaction_rule: YGL255W or YLR130C
+    - annotation: !!omap
+      - metanetx.reaction: MNXR105278
+      - sbo: SBO:0000655
+    - confidence_score: 1
+  - !!omap
+    - id: r_4592
+    - name: Mg(2+) transport
+    - metabolites: !!omap
+      - s_0794: 1
+      - s_0796: -1
+      - s_4013: 1
+      - s_4204: -1
+    - lower_bound: -1000
+    - upper_bound: 1000
+    - gene_reaction_rule: GR191W or YOL130W
+    - annotation: !!omap
+      - metanetx.reaction: MNXR101553
+      - sbo: SBO:0000655
+    - confidence_score: 1
```

Comment:
if some of these rxns have genes associated shouldn't they have a confidence score of 2?